acceptance/lib/puppet/acceptance/temp_file_utils.rb

```diff
@@ -140,16 +140,25 @@ def chmod(host, mode, path)
       # value containing only the file mode, excluding the type, e.g
       # S_IFDIR 0040000
       def stat(host, path)
-        stat_command = case host['platform']
-                       when /osx/
-                         "stat -f '%Su:%Sg:%p'"
-                       else
-                         "stat --format '%U:%G:%a'"
-                       end
-
-        permissions = on(host, "#{stat_command} #{path}").stdout.chomp
-        owner, group, mode = permissions.split(':')
+        owner, group, mode =
+          if host['platform'] =~ /solaris-10/
```

Comment:
This is probably a bad idea, so we could also just disable the 2 tests that use it on Solaris 10 for now.